desktop_version/src/UtilityClass.cpp

```diff
@@ -1,67 +1,63 @@
 #define HELP_DEFINITION
 #include "UtilityClass.h"
 
-#include <cctype>
 #include <SDL.h>
 #include <sstream>
 
-/* Used by UtilityClass::GCString to generate a button list */
```

Comment:
Might be a good comment to keep, but we're probably dumping a lot of this when input gets redone anyway...

Reply:
Yeah, it'll probably get axed anyway. If you really want me to keep this, I can; otherwise I don't see the point.